awx/ui_next/src/routeConfig.jsx

```diff
@@ -22,8 +22,11 @@ import Users from './screens/User';
 import WorkflowApprovals from './screens/WorkflowApproval';
 import { Jobs } from './screens/Job';
 
-function getRouteConfig() {
-  return [
+function getRouteConfig(me = {}) {
+  if (!me) {
+    return [];
+  }
```

Comment:
```suggestion
```

@nixocio We should delete this conditional now that we're using the optional chaining in the conditionals below

Reply:
done.